apps/react/src/features/studios/components/StudioDetail/StudioCard.tsx

```diff
@@ -0,0 +1,33 @@
+import { memo, FC } from 'react';
+import { useParams } from 'react-router-dom';
+import { Paper, Typography } from '@mui/material';
+
+/** Card with studio data. */
+const StudioCardComponent: FC = () => {
+	const { id } = useParams<{ id: string; }>();
+
+	return (
+		<Paper
+			elevation={3}
+			sx={{
+				padding: '16px',
```

Comment:
Please, use spacing tokens provided by MUI. You can read more here: https://mui.com/material-ui/customization/spacing/

No need to use `theme.spacing` directly, you can use it like this:

```suggestion
				padding: 4,
```